### ISSUE TITLE
Mac OSX Passcommand syntax issue

### ISSUE THREAD

**Issue by ugaman42** [borgbackup/borg#8174]
Does anyone know what the updated syntax should be for using BORG_PASSCOMMAND with the Mac OS keychain? I am receiving a "base64: invalid option -- w" message when attempting to run the command that should generate the "passphrase" in my keychain from the help docs for using this string found in the Borg FAQ's:

`security add-generic-password -D secret -U -a $USER -s borg-passphrase -w $(head -c 32 /dev/urandom | base64 -w 0)   `

It appears there may be an issue with the base64 syntax after the pipe at the end. This is on a silicon Mac. 

link to FAQ: https://borgbackup.readthedocs.io/en/stable/faq.html

**Comment by ThomasWaldmann** [comment]
@ugaman42 I transferred this issue from community repo to here.

**Comment by ugaman42** [comment]
> @ugaman42 I transferred this issue from community repo to here.

Thank you. I will create the pr. Would you like me to specify for silicon Mac’s? I’m not sure if the -w 0 works on intel based or not. I do not have one to test on.

**Comment by ugaman42** [comment]
> Solution: https://stackoverflow.com/questions/46463027/base64-doesnt-have-w-option-in-mac
> 
> TL;DR: just remove "-w 0".
> 
> Can you make a PR to fix the docs?

I’m new to GitHub and PR’s in general. It’s not as straight forward as I thought and I’d like to read through the borg PR docs and GitHub PR docs to ensure I do it correctly. It will have to wait until after work, but I’ll get to it. I’d like to learn it. Thank you.

**Comment by ThomasWaldmann** [comment]
Guess the best approach is to start with a PR against 1.2-maint branch (1.2.x releases are created from there) and after that is merged, forward port the change to 1.4-maint and master branches.

I assume you use a recent macOS release and also that the same macOS release will not have different command options on intel machines.
